docs/index.rst

```diff
@@ -1,43 +1,41 @@
-.. scrapy-poet documentation master file, created by
-   sphinx-quickstart on Tue Apr 28 16:45:10 2020.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
-
 =========================
 scrapy-poet documentation
 =========================
 
-scrapy-poet easily integrates Page Objects created using `web-poet`_ with
-Scrapy through the configuration of a dependency injection middleware.
+scrapy-poet allows to use `web-poet`_ Page Objects with Scrapy.
+
+web-poet_ defines a standard for writing reusable and portable
+extraction and crawling code; please check its docs_ to learn more.
```

Comment:
Does it allow crawling?

Reply:
What I'm thinking about here is that we should be standardizing not only `to_item()`method, but methods related to crawling as well - e.g. pagination API, or API for lists of items. Crawling code can then use these building blocks.